setup.py

```diff
@@ -62,4 +102,7 @@
         'Topic :: System :: Archiving :: Compression',
     ],
     python_requires='>=3.6',
+    setup_requires=[
+        "pkgconfig",
+    ],
```

Comment:
did you try how that behaves?

i tried this for `borgbackup`, but the problem is that `setup.py` already needs `pkgconfig`  before `setup()` is executed (and could install `pkgconfig` due to `setup_requires`).